features/comment.feature

```diff
@@ -1,41 +0,0 @@
-Feature: Different users can comment on an App
```

Comment:
Also, I'm not sure what's going on, but why are there blocks of red for all the other feature files? Does this mean they are deleted?

Reply:
I deleted the files that had failing tests and were not in the files I worked on to pass Travis. They should all be merged back when we submit all PRs.